frontend/app/player/Player.ts

```diff
@@ -151,21 +158,21 @@ export default class Player extends MessageDistributor {
     }
   }
 
-  jump(time = getState().time, index) {
-    const { live } = getState();
-    if (live) return;
+  jump(time = getState().time, index: number) {
+    const { live, liveTimeTravel, endTime } = getState();
+    if (live && !liveTimeTravel) return;
     
     if (getState().playing) {
       cancelAnimationFrame(this._animationFrameRequestId);
       // this._animationFrameRequestId = requestAnimationFrame(() => {
         this._setTime(time, index);
         this._startAnimation();
-        update({ livePlay: time === getState().endTime });
+        update({ livePlay: Math.abs(time - endTime) < 500 });
```

Comment:
Please add a comment about the purpose of updating  not faster than 0.5 sec . You will yobneshsya later to understand what and why otherwise.